pkg/capacity/constants.go

```diff
@@ -0,0 +1,4 @@
+package capacity
```

Comment:
Tiny Nit: Please add the Apache 2 License header here

Reply:
np will do

Reply:
> Thanks so much for this PR @fnickels! This works great and is a useful addition. A couple tiny nits but otherwise LGTM. Feel free to add yourself to the contributors list in the README.